mlx/backend/metal/indexing.cpp

```diff
@@ -1,7 +1,7 @@
 // Copyright © 2023 Apple Inc.
-
 #include <algorithm>
 #include <cassert>
+#include <iostream> // TODO
```

Comment:
I know this has a TODO, just wanted to point it out before a merge

Reply:
Ty! I knew that TODO was a good idea :)